### ISSUE TITLE
Please Release a New Version

### ISSUE THREAD

**Issue by eljobe** [unittest-cpp/unittest-cpp#140]
Sorry to file an issue for what is really more of a request, but I didn't see another forum for discussing this library's development schedule and release process.

I am really excited to see the support for CMake added in this change: unittest-cpp/unittest-cpp@e88121f

I would like to see it become part of a stable release of the library including having it installed by Homebrew on Mac OS X.  Does this project also create the Homebrew releases when it is time?

**Comment by pjohnmeyer** [comment]
As of today I am starting on getting 2.0 released; you can expect it in the next few weeks. I have in the past and will hopefully remember to issue a pull request to Homebrew.

**Comment by grahamreeds** [comment]
I am still a bit mystified as to what 2.x is going to give us over 1.7.

Usually you change the major number if you are going to make breaking
changes and I believe that would be a bad idea. You are threatening to
strand a lot of people on 1.6.x.

The only thing I could see that might warrant a 2.x is a total rewrite
making it less macro based but since c++ doesn't have reflection I think
(like so many other attempts) that it would be doomed to failure.

GR

**Comment by pjohnmeyer** [comment]
@grahamreeds :

[Back in January](http://unittest-cpp.github.io/2016/01/28/version-1-5-and-beyond.html) I announced that the project would start using Semantic Versioning after 1.5.0.

In July you wrote [this issue](https://github.com/unittest-cpp/unittest-cpp/issues/119) which finally gave me the driver to make the change suggested in [this request](https://github.com/unittest-cpp/unittest-cpp/issues/7). That change, per Semantic Versioning, required me to change the major version number. I [wrote about this in August](http://unittest-cpp.github.io/2016/08/25/version-2-preview.html), explaining why it was a breaking change, and explaining how minor the changes were. Your issue was labeled 2.0.0 around that same time.

In the one month that [the pull request](https://github.com/unittest-cpp/unittest-cpp/pull/120) remained open, nobody raised any concerns. In the nearly five months between its merge and release, nobody raised any concerns.

In short, 2.0 implements an enhancement requested before the project was even on GitHub, in order to fix an issue **you** requested be fixed, in a manner which will not break 99% of users, and will possibly make tests run faster for others, after months of silence from the community on the matter.

I would be shocked if these changes "strand" anybody, but they **are** _potentially_ breaking. It would be incorrect to call this 1.7, and I have no interest in rolling the changes back as they are architecturally sound changes. Everyone is welcome to use 1.6.x for as long as they like -- it's not going anywhere -- but the project is moving forward.
